index.html

```diff
@@ -5,11 +5,61 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Portfolio</title>
+    <link href="https://fonts.cdnfonts.com/css/poppins" rel="stylesheet" />
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
-    <header class="main-header">
-      <a href=""><logo>Noel Bryant</logo></a>
-    </header>
+    <div class="headline-container">
+      <header>
+        <nav class="navbar">
+          <!-- <div class="toolbar"> -->
+          <div class="logo">
+            <a href="">
+              <img src="assets/images/logo.png" alt="my awesome logo"
+                />
+            </a>
+          </div>
+          <div class="tertiary-enabled">
+            <img
+              src="assets/images/mobile-menu.svg"
+              alt="menu icon"
+              class="mobilenav"
+              />
+            <!-- </div> -->
+          </div>
+        </nav>
+      </header>
+
+      <section class="headline">
+        <h1>I’m Noel Bryant from Kenya. Glad to see you!</h1>
+        <p>
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hesitate to contact
+          me.
+        </p>
+        <div class="social-media">
+          <p>Let's connect</p>
```

Comment:
- [ ] This is a link. Please use the anchor tag. This is because all links in HTML are anchor tags and apart from being semantically correct, the `a` tag will let you connect to other sections of the website, using href. 😺